tools/mc-java-protoc/src/main/java/io/spine/tools/mc/java/protoc/method/MethodGen.java

```diff
@@ -70,13 +71,23 @@ public static MethodGen instance(SpineProtocConfig spineProtocConfig) {
         Classpath classpath = spineProtocConfig.getClasspath();
         ExternalClassLoader<MethodFactory> classLoader =
                 new ExternalClassLoader<>(classpath, MethodFactory.class);
-        AddMethods config = spineProtocConfig.getAddMethods();
         ImmutableList.Builder<CodeGenerationTask> tasks = ImmutableList.builder();
-        if (isNotDefault(config.getUuidFactory())) {
-            tasks.add(new GenerateUuidMethods(classLoader, config.getUuidFactory()));
+        if (spineProtocConfig.hasUuids()) {
+            List<MethodFactoryName> methodFactoryNames = spineProtocConfig.getUuids()
+                                                                          .getMethodFactoryList();
+            methodFactoryNames
+                    .stream()
+                    .map(name -> new GenerateUuidMethods(classLoader, name))
+                    .forEach(tasks::add);
         }
-        for (ConfigByPattern byPattern : config.getFactoryByPatternList()) {
-            tasks.add(new GenerateMethods(classLoader, byPattern));
+        for (ForMessages forMessages : spineProtocConfig.getMessagesList()) {
+            Pattern pattern = forMessages.getPattern();
+            forMessages.getGenerateMethodsList()
+                       .stream()
+                       .map(generate -> new GenerateMethods(
```

Comment:
The snippet `generate -> new GenerateMethods(...)` looks as a popular thing in this update. I commented on this already in relation to one file, but we're having a number of similar looking blocks, where we tend to tell similar things. We need to generalize it somehow.

Reply:
Those are all different types though they do look similar.

I find it handy to name them in such a cohesive way.

Let's discuss this in person if necessary.